server/comments/views.py

```diff
@@ -49,9 +46,6 @@ class GetRemoteCommentView(generics.ListCreateAPIView):
     serializer_class = CommentSerializer
 
     def post(self, request, *args, **kwargs):
-        post_id = self.kwargs['post_id']
```

Comment:
just a question, are you using the id provided in the passed in object?

Reply:
no. there's no point. I'm gonna just pass the entire thing in and let the other team handle it.